src/views/registration/attendees/fields/select.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Descriptio
+ *
+ * @version TBD
+ *
+ */
+$required      = isset( $field->required ) && 'on' === $field->required ? true : false;
+$field         = (array) $field;
+$attendee_id   = null;
+$value         = '';
+$is_restricted = false;
+$slug          = $field['slug'];
+$options       = null;
+$field_name    = 'tribe-tickets-meta[' . $attendee_id . '][' . esc_attr( $slug ) . ']';
+
+if ( isset( $field['extra'] ) && ! empty( $field['extra']['options'] ) ) {
+	$options = $field['extra']['options'];
```

Comment:
lets use `Tribe__Utils__Array::get( array( 'extra', 'options' ) )` to avoid conditional